common/script_values/wc_men_at_arms_values.txt

```diff
@@ -0,0 +1,148 @@
+﻿######################
+# Men-at-Arms Values #
+######################
+
+#########
+# Costs #
+#########
+
+@recr_to_maint_factor = 0.004
+@high_maint_mult = 3
+
+###Vanilla Refrence for Camels/Elekks
+@light_cavalry_recruitment_cost = 85
+@light_cavalry_low_maint_cost = 0.35
+@light_cavalry_high_maint_cost = @[light_cavalry_low_maint_cost * high_maint_mult]
+
+@heavy_cavalry_recruitment_cost = 200
+@heavy_cavalry_low_maint_cost = 0.7
+@heavy_cavalry_high_maint_cost = @[heavy_cavalry_low_maint_cost * high_maint_mult]
+
+### Base Cost Values
+@massive_infantry_recruitment_cost = 115
+@massive_infantry_low_maint_cost = @[massive_infantry_recruitment_cost * recr_to_maint_factor]
+@massive_infantry_high_maint_cost = @[massive_infantry_low_maint_cost * high_maint_mult]
+
+@caster_recruitment_cost = 90
+@caster_low_maint_cost = @[caster_recruitment_cost * recr_to_maint_factor]
+@caster_high_maint_cost = @[caster_low_maint_cost * high_maint_mult]
+
+@light_aerial_recruitment_cost = 75
+@light_aerial_low_maint_cost = @[light_aerial_recruitment_cost * recr_to_maint_factor]
+@light_aerial_high_maint_cost = @[light_aerial_low_maint_cost * high_maint_mult]
+
+@massive_aerial_recruitment_cost = 200
+@massive_aerial_low_maint_cost = @[massive_aerial_recruitment_cost * recr_to_maint_factor]
+@massive_aerial_high_maint_cost = @[massive_aerial_low_maint_cost * high_maint_mult]
+
+### Base MAA
+massive_infantry_recruitment_cost = @[massive_infantry_recruitment_cost]
+massive_infantry_low_maint_cost = @[massive_infantry_low_maint_cost]
+massive_infantry_high_maint_cost = @[massive_infantry_high_maint_cost]
+
+caster_recruitment_cost = @[caster_recruitment_cost]
+caster_low_maint_cost = @[caster_low_maint_cost]
+caster_high_maint_cost = @[caster_high_maint_cost]
+
+light_aerial_recruitment_cost = @[light_aerial_recruitment_cost]
+light_aerial_low_maint_cost = @[light_aerial_low_maint_cost]
+light_aerial_high_maint_cost = @[light_aerial_high_maint_cost]
+
+massive_aerial_recruitment_cost = @[massive_aerial_recruitment_cost]
+massive_aerial_low_maint_cost = @[massive_aerial_low_maint_cost]
+massive_aerial_high_maint_cost = @[massive_aerial_high_maint_cost]
+
+### Cultural MAA
+#Add custom costs for each cultural maa here
+### Regional MAA
+
+### Regional MAA
+camel_rider_recruitment_cost = @[light_cavalry_recruitment_cost + 15]
+camel_rider_low_maint_cost = @[light_cavalry_low_maint_cost * 0.8]
+camel_rider_high_maint_cost = @[light_cavalry_high_maint_cost * 0.8]
+
+war_elephant_recruitment_cost = @[heavy_cavalry_recruitment_cost * 2]
+war_elephant_low_maint_cost = @[heavy_cavalry_low_maint_cost * 1.2]
+war_elephant_high_maint_cost = @[heavy_cavalry_high_maint_cost * 1.4]
```

Comment:
Why it's here and not in the original file?

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/105491051-27c2de00-5ccf-11eb-83cd-768ebc56b2a9.png)

</details>

You even added `light_cavalry_recruitment_cost` in this file just for that creating two duplicates of `light_cavalry_recruitment_cost`.